tests/conftest.py

```diff
@@ -307,85 +290,281 @@ def simple_inventory_node() -> None:
+def simple_parameter_dict() -> dict:
+    ret_dict = {"node": ["Parameter"], "key": "update_frequency", "value": 1000.0, "unit": "1/ns"}
+    return ret_dict
 
-    return my_property
+
+@pytest.fixture(scope="function")
+def simple_algorithm_node() -> cript.Algorithm:
+    algorithm = cript.Algorithm("mc_barostat", "barostat")
+    return algorithm
 
 
-# ---------- Supporting Nodes ----------
 @pytest.fixture(scope="function")
-def simple_file_node() -> cript.File:
-    """
-    simple file node with only required arguments
-    """
-    my_file = cript.File(source="https://criptapp.org", type="calibration", extension=".csv", data_dictionary="my file's data dictionary")
+def simple_algorithm_dict() -> dict:
+    ret_dict = {"node": ["Algorithm"], "key": "mc_barostat", "type": "barostat"}
+    return ret_dict
 
-    return my_file
+
+@pytest.fixture(scope="function")
+def simple_reference_node() -> cript.Reference:
+    title = "Multi-architecture Monte-Carlo (MC) simulation of soft coarse-grained polymeric materials: "
+    title += "SOft coarse grained Monte-Carlo Acceleration (SOMA)"
+
+    reference = cript.Reference(
+        "journal_article",
+        title=title,
+        authors=["Ludwig Schneider", "Marcus Müller"],
+        journal="Computer Physics Communications",
+        publisher="Elsevier",
+        year=2019,
+        pages=[463, 476],
+        doi="10.1016/j.cpc.2018.08.011",
+        issn="0010-4655",
+        website="https://www.sciencedirect.com/science/article/pii/S0010465518303072",
+    )
+    return reference
+
+
+@pytest.fixture(scope="function")
+def simple_reference_dict() -> dict:
+    ret_dict = {
+        "node": ["Reference"],
+        "type": "journal_article",
+        "title": "Multi-architecture Monte-Carlo (MC) simulation of soft coarse-grained polymeric materials: SOft coarse grained Monte-Carlo Acceleration (SOMA)",
+        "authors": ["Ludwig Schneider", "Marcus Müller"],
+        "journal": "Computer Physics Communications",
+        "publisher": "Elsevier",
+        "year": 2019,
+        "pages": [463, 476],
+        "doi": "10.1016/j.cpc.2018.08.011",
+        "issn": "0010-4655",
+        "website": "https://www.sciencedirect.com/science/article/pii/S0010465518303072",
+    }
+    return ret_dict
 
 
 @pytest.fixture(scope="function")
 def simple_citation_node(simple_reference_node) -> cript.Citation:
-    """
-    minimal citation node
-    """
-    my_citation = cript.Citation(type="derived_from", reference=simple_reference_node)
+    citation = cript.Citation("reference", simple_reference_node)
+    return citation
+
 
-    return my_citation
+@pytest.fixture(scope="function")
+def simple_citation_dict(simple_reference_dict) -> dict:
+    ret_dict = {"node": ["Citation"], "reference": simple_reference_dict, "type": "reference"}
+    return ret_dict
 
 
 @pytest.fixture(scope="function")
 def simple_quantity_node() -> cript.Quantity:
```

Comment:
I think it might be a good idea to get some docstrings for these functions

Reply:
I don't know what to write in those to be hones.
They are very short functions with only one active line of code each.
If you want to write some, sure go ahead.